doc/docs.md

```diff
@@ -2049,6 +2050,88 @@ fn init() {
 The `init` function cannot be public - it will be called automatically. This feature is
 particularly useful for initializing a C library.
 
+### Module/package management
+
+Briefly:
+
+```powershell
+v [module option] [param]
+```
+
+###### module option's:
```

Comment:
No need for `'` here - that is the possessive form, meaning "belonging to the option", where you just want to say there are multiple options.

Reply:
Looks fine other than that.

Reply:
Silly question about Github:
 I can no longer commit to koplenov:patch-1 because I deleted that repository of mine?

So, in order to fix " ' " I have to do everything all over again?

 (；ω；)

Reply:
Ugh.  If you've deleted that repository I don't know if a merge will work, so you may have to make the changes again anyway.  I don't know if Github has cached things so that it will work or not.

I would vote to try to get this merged, then we can remove the `'` as a separate thing.

Reply:
The last way looks simpler